app/components/Views/WalletActions/WalletActions.test.tsx

```diff
@@ -180,4 +187,48 @@ describe('WalletActions', () => {
 
     expect(mockNavigate).toHaveBeenCalled();
   });
+  it('should disable buttons when the account cannot sign transactions', () => {
```

Comment:
nit: I know the rest of the file is using the previous convention but that shouldn't stop us of using the new one

```suggestion
  it('disables action buttons when the account cannot sign transactions', () => {
```